controller/test/sensor_tests/sensors_test.cpp

```diff
@@ -104,24 +104,21 @@ TEST(SensorTests, FullScaleReading) {
   }
 }
 
-// These tests expect Venturi Diamters of 14 and 5.5 mm.
-// If the Default PressureSensors::DEFAULT_VENTURI_PORT_DIAM and
-// DEFAULT_VENTURI_CHOKE_DIAM are changed from this, the tests will fail unless
-// you update the expected values accordingly.
+// These tests expect Venturi Diamters of 15.05mm and 5.5mm with a "discharge
+// coefficient" (i.e. fudge factor) of 0.97.  If VENTURI_PORT_DIAM or
```

Comment:
Its not really a fudge factor. 1.00 implies "perfect" energy recovery, i.e., no mechanical losses. 0.97 accounts for these losses. It is not the optimal fitting parameter (i.e., we did not develop the parameter to minimize R^2) rather it is the ISO recommendation for venturis with flow at Reynolds number ~10^4.